### DISCUSSION TITLE
Show number of unresolved hidden conversations

### DISCUSSION THREAD

**Discussion by ChrisLabattD2L** [community/community#75820] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

The `x hidden conversations` section (see screenshot) in a Pull Request is a good way to reduce the number of conversations that appear when there are a lot, but it would be helpful if it also told you how many of the hidden conversations were resolved/unresolved before having to expand them when looking back at a Pull Request to make sure everything is resolved before approving or merging it. Or grouping together resolved comments into the hidden section and having the unresolved ones not included in it.

![image](https://github.com/community/community/assets/71081906/173a0114-2051-43ae-b8a3-35fad0b71a5d)

**Comment by csengineer1990** [comment]
Thanks for your feedback! That's a solid point about the hidden conversations in Pull Requests. Including information about how many of them are resolved or unresolved before expanding would indeed be super helpful for a quick overview.

**Reply by medmunds** [reply]
Although showing the number of unresolved hidden conversations would be an improvement, it would be even more helpful to *not hide unresolved conversations* in the first place. That's come up repeatedly as #11842, #112603, and #130618 (and possibly others I haven't found).

**Comment by ChrisMacGregorONI** [comment]
https://github.com/orgs/community/discussions/130618#discussioncomment-12105463
